pink/src/redis_cli.cc

```diff
@@ -534,7 +534,7 @@ int redisvFormatCommand(std::string *cmd, const char *format, va_list ap) {
               memcpy(_format, c, _l);
               _format[_l] = '\0';
 
-              int n = vsnprintf(buf, REDIS_MAX_MESSAGE, _format, _cpy);
+              int n = vsnprintf(buf, sizeof(buf), _format, _cpy);
```

Comment:
Btw, what's wrong with just using the cpp function `snprintf`?

Reply:
Nothing wrong, after reading the docs it seems like it is an easier API to use.